Bundle/CoreBundle/Twig/Extension/RoutingExtention.php

```diff
@@ -43,8 +43,15 @@ public function getPath($name, $parameters = [], $relative = false)
                 $params['entityId'] = $parameters['entityId'];
                 unset($parameters['entityId']);
             }
-            $page = $this->pageHelper->findPageByParameters($params);
-            $parameters['url'] = $page->getReference($this->request->getLocale())->getUrl();
+
+            try {
+                $page = $this->pageHelper->findPageByParameters($params);
+                $parameters['url'] = $page->getReference($this->request->getLocale())->getUrl();
+            } catch (\Exception $e) {
+                error_log($e->getMessage());
+                $parameters['url'] = '410';
```

Comment:
Would it be better to `$em->getRepository('VictoireTwigBundle:ErrorPage')findOneByErrorCode(410)` ?